README.md

```diff
@@ -213,6 +213,27 @@ in place of a file being edited.)
 /usr/include/dmd/phobos/std/conv.d  f   9494
 ```
 
+## Find usage of symbol at cursor
+```dcd-client --localUsage -c 123```
+
+The "--localUsage" or "-u" flags cause the client to instruct the server
+to return all the local usages of the symbol located at the given cursor position.
+
+#### Output format
+When usages exist, if the source symbol is an identifier (a type, a variable name, etc.) and if the symbol is not ambiguous then
+the first line contains the location of the symbol (a file name or literally _stdin_), a tab then the offset to the symbol declaration,
```

Comment:
> Because filenames can contain spaces they not tabs

A yes, I forgot. Ok then, using tabs is fine.

> Again, because of the compatibility with my IDE, this can't be done right now.
And I don't think that the length is useful. Editors and IDE have methods to do that.
So maybe later

Alright.